web/src/env.js

```diff
@@ -0,0 +1 @@
+export default {REST_Url: 'http://localhost:8991/'}
```

Comment:
sure, temporary solution once again.  Don't worry as it's easy to completely rewrite the code at anytime.  You also mentioned that this at somepoint might just be threads inside kelp instead of launched tasks, so I didn't want to go through the trouble of making it configurable and perfect when things may change, but I can fix that if we have decided for sure how it's going to work.

Reply:
ok fair enough, let's leave this as-is.